### DISCUSSION TITLE
Caching s3 images in vercel

### DISCUSSION THREAD

**Discussion by collegewap** [vercel/vercel#7736] [Help]
Here is the screenshot of my usage dashboard. I've used Cloudinary during the highlighted period and as you could see 60-70% of the requests were served from cache. In the remaining period, I have used Amazon s3 and only 12% of the requests are cached. Can we cache s3 images as well? The only difference here is where the image is stored. No other code changes.

![image](https://user-images.githubusercontent.com/9767383/165970611-46edfc7e-b26e-4670-9ca9-b4dc8f5fb279.png)

**Comment by sladg** [comment]
Hey! Looking into this as well. It seems like S3 assets (even when WORM is used) are not cached.
Should we set specific headers in metadata to allow for caching? Cannot really find in-depth info on this.
